pkg/cluster/internal/providers/docker/network.go

```diff
@@ -69,8 +75,19 @@ func ensureNetwork(name string) error {
 	// If it is, make more attempts below
 	if isIPv6UnavailableError(err) {
 		// only one attempt, IPAM is automatic in ipv4 only
-		return createNetwork(name, "")
-	} else if !isPoolOverlapError(err) {
+		return createNetworkNoDuplicates(name, "")
```

Comment:
why don't we check ` if isPoolOverlapError(err) ` like in the other places?

Reply:
ahh, just read IPAM is automatic, hope ipam handles overlapping too

Reply:
When we pass "" for the IPv6 subnet to this method we don't enable IPv6, in IPv4 we let docker's internal allocator handle it. This is from before the PR.